source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+---
```

Comment:
The docs sidebar is not shown on this page. We have an aside that checks the URL to see which sidebar to load, it should include /projects/